README.md

```diff
@@ -133,6 +130,14 @@ Erigon (https://docs.gnosischain.com/category/step--3---run-consensus-client).
 
 Running `make help` will list and describe the convenience commands available in the [Makefile](./Makefile).
 
+### Upgrading from 3.0 to 3.1
+
+When running Erigon 3.1+, your snapshot files will be upgraded automatically to a new file naming scheme. It's recommended that you take a backup or filesystem snapshot of your datadir before upgrading.
+
+The downloader component in Erigon 3.1 will check the file data of snapshots when `--downloader.verify` is provided. Incorrect data will be repaired.
+
+A new `snapshots reset` subcommand is added, that lets you trigger Erigon to perform an initial sync on the next run. Use this if your datadir is corrupted, or your client is unable to obtain missing snapshot data due to having committed to a snapshot that is no longer available. It will remove any locally generated files, and your chain data. Pass `--local=false` to force a resync but keep files that aren't in the latest snapshot.
```

Comment:
`that lets you trigger Erigon to perform an initial sync on the next run` -- sounds like it's sync from scratch. Maybe we should call it "initial sync with repair and reuse of existing files".

Reply:
Thanks, I am definitely trying to convey that files are preserved. I've amended the sentence.